app/webpack/stylesheets/open_studios_subdomain/_container_header.scss

```diff
@@ -24,8 +23,13 @@
 }
 .container-header__title {
   text-align: center;
-  margin: 0;
+  margin: 0 0 8px 5px;
   flex: 1;
+  font-size: 2em;
+  @media screen and (max-width: $screen-sm-max) {
+    font-size: 1em;
```

Comment:
I think we can get away with a little bigger - maybe 1.6em - I wish I'd done better about font sizes.  I hate `em` and would prefer `rem` but I can't make a good argument for it since the font story in this app is a mess. :D